package.json

```diff
@@ -121,6 +121,7 @@
     "realm": "^12.3.0",
     "reassure": "^0.10.1",
     "sanitize-html": "^2.11.0",
+    "tailwind-merge": "^2.2.0",
```

Comment:
Please remove this and use the classnames package instead. I'm pretty sure it does exactly the same thing.